src/models/searchservice/response/directanswer/BaseFieldValueDirectAnswer.ts

```diff
@@ -1,14 +1,15 @@
 import { DirectAnswer } from './DirectAnswer';
 import { DirectAnswerType } from './DirectAnswerType';
 import { BuiltInFieldType } from './BuiltInFieldType';
-import { Result } from './Result';
+import { Result } from '../Result';
+import { EnumOrLiteral } from '../../../utils/EnumOrLiteral';
 
 /**
  * A direct answer where the answer came from a field from the knowledge graph.
  *
  * @public
  */
-export interface FieldValueDirectAnswer<T = unknown> extends DirectAnswer<T> {
+export interface BaseFieldValueDirectAnswer<T = unknown> extends DirectAnswer<T> {
   /** {@link DirectAnswerType}.FieldValue. */
   type: DirectAnswerType.FieldValue,
```

Comment:
I know we said previously we didn't care about removing duplicate properties, but I played around with it a bit more and I think it simplifies stuff a good bit so maybe we could do that here?

Reply:
Yea, it would certainly simplify things. I can remove duplicate for direct answer interfaces

Reply:
updated!